doc/source/release-notes/Percona-XtraDB-Cluster-5.7.21-29.26.rst

```diff
@@ -0,0 +1,67 @@
+.. rn:: 5.7.21-29.26
+
+=====================================
+Percona XtraDB Cluster 5.7.21-29.26
+=====================================
+
+Percona is glad to announce the release of
+Percona XtraDB Cluster 5.7.21-29.26 on March 02, 2018.
+Binaries are available from the `downloads section
+<http://www.percona.com/downloads/Percona-XtraDB-Cluster-57/>`_
+or from our :ref:`software repositories <install>`.
+
+Percona XtraDB Cluster 5.7.20-29.24 is now the current release,
+based on the following:
+
+* `Percona Server for MySQL 5.7.21 <https://www.percona.com/doc/percona-server/5.7/release-notes/Percona-Server-5.7.21-20.html>`_
+
+* Galera Replication library 3.23
+
+* Galera/Codership WSREP API Release 5.7.21
+
+Starting from now, Percona XtraDB Cluster issue
+tracking system was moved from launchpad to
+`JIRA <https://jira.percona.com/projects/PXC>`_.
+All Percona software is open-source and free.
+
+Fixed Bugs
+==========
+* :jirabug:`PXC-2039`: Node consistency was compromised for
+  ``INSERT INTO ... ON DUPLICATE KEY UPDATE`` workload
+  because the regression introduced in |PXC| :rn:`5.7.17-29.20`
+  made it possible to abort local transactions without further
+  re-evaluation in case of a lock conflict.
+
+* :jirabug:`PXC-2054` Redo optimized DDL operations (like sorted index
```

Comment:
though this was done for redo optimized ddl, xb option to blocks all ddl through --lock-ddl option. We want to mention redo optimized ddl operation influenced it but with a hint that all ddl are blocked.